Framework/MDAlgorithms/test/IntegratePeaksMD2Test.h

```diff
@@ -180,6 +180,128 @@ class IntegratePeaksMD2Test : public CxxTest::TestSuite {
     TS_ASSERT(algF.isExecuted());
   }
 
+  //-------------------------------------------------------------------------------
+  /** Setup simple algorithm*/
+  static void simpleRun(bool shouldPass, size_t numberValuesPeakRadius, size_t numberValuesBkgInnerRadius,
+                        size_t numberValuesBkgOuterRadius, bool cyl = false, bool ellip = false) {
+    createMDEW();
+    /* Create 3 overlapping shells so that density goes like this:
+     * r < 1 : density 1.0
+     * 1 < r < 2 : density 1/2
+     * 2 < r < 3 : density 1/3
+     */
+    addPeak(1000, 0., 0., 0., 1.0);
+    addPeak(1000 * 4, 0., 0., 0.,
+            2.0); // 8 times the volume / 4 times the counts = 1/2 density
+    addPeak(1000 * 9, 0., 0., 0.,
```

Comment:
As these tests only check whether the algorithm throws (or not!) I don't think we actually need to generate data?